filter_v2_exomes_to_unique.py

```diff
@@ -0,0 +1,58 @@
+import argparse
+import logging
+
+import hail as hl
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("filter_v2_exome_variants")
+logger.setLevel(logging.INFO)
+
+def main(args):
+    logger.info("Loading variant table, v2 genomes, v3 variants, and v2 liftover...")
+    samples_with_variants = hl.read_table(args.sample_with_variants_path)
+    original_count = samples_with_variants.count()
+    v2_genomes = hl.read_table("gs://gcp-public-data--gnomad/release/2.1.1/ht/genomes/gnomad.genomes.r2.1.1.sites.ht")
+    v3_variants = hl.read_table("gs://gcp-public-data--gnomad/release/3.1.1/ht/genomes/gnomad.genomes.v3.1.1.sites.ht")
+    v2_liftover = hl.read_table("gs://gcp-public-data--gnomad/release/2.1.1/liftover_grch38/ht/exomes/gnomad.exomes.r2.1.1.sites.liftover_grch38.ht")
+    v2_liftover = v2_liftover.key_by("original_locus", "original_alleles")
+    v2_liftover_index = v2_liftover[samples_with_variants.locus, samples_with_variants.alleles]
+    samples_with_variants = samples_with_variants.annotate(liftover_locus = v2_liftover_index.locus, liftover_alleles = v2_liftover_index.alleles)
+    logger.info("Filtering variants not found in v2 genomes or v3_non_v2...")
+    idx = v2_genomes.freq_meta.collect()[0].index({'group': 'adj'})
+    samples_with_variants = samples_with_variants.annotate(v2_genomes_adj_freq = v2_genomes[samples_with_variants.locus,samples_with_variants.alleles].freq[idx])
+    v2_genomes_filtering_expr = (samples_with_variants.v2_genomes_adj_freq.AC<1) | ~hl.is_defined(samples_with_variants.v2_genomes_adj_freq.AC)
+    samples_with_variants = samples_with_variants.filter(v2_genomes_filtering_expr)
+    idx = v3_variants.freq_meta.collect()[0].index({'group': 'raw', 'subset': 'non_v2'})
```

Comment:
Do we want raw here, or adj? If adj:
```suggestion
    freq_adj_idx = v3_genomes_ht.freq_meta.collect()[0].index({'group': 'adj', 'subset': 'non_v2'})
```

Reply:
good catch!